content/40.inequality.md

```diff
@@ -182,31 +253,32 @@ This approach would carry its own ethical concerns, including the fact that many
 As the pandemic has progressed, it has become clear that ICU beds and ventilators are not the only limited resources that needs to be allocated, and, in fact, the survival rate for patients who receive mechanical ventilation is lower than these discussions would suggest [@doi:10.1136/medethics-2020-106460].
 Allocation of interventions that may reduce suffering, including palliative care, has become critically important [@doi:10.1136/medethics-2020-106460; @doi:10.1080/15265161.2020.1788663].
 The ambiguities surrounding the risks and benefits associated with therapeutics that have been approved under emergency use authorizations also present ethical concerns related to the distribution of resources [@doi:10.1080/15265161.2020.1795538].
-For example, remdesivir, discussed above, is currently available for the treatment of COVID-19 under compassionate use guidelines and through expanded access programs, and in many cases has been donated to hospitals by Gilead [@doi:10.1016/j.mayocp.2020.06.016; @doi:10.1080/15265161.2020.1795529].
+For example, for much of 2020, remdesivir was available for the treatment of COVID-19 under compassionate use guidelines and through expanded access programs, and in many cases has been donated to hospitals by the manufacturer, Gilead [@doi:10.1016/j.mayocp.2020.06.016; @doi:10.1080/15265161.2020.1795529].
+Remdesivir is now the only FDA-approved treatment for COVID-19.<!--To Do: cite therapeutics-->
 Regulations guiding the distribution of drugs in situations like these typically do not address how to determine which patients receive them [@doi:10.1080/15265161.2020.1795529].
-Prioritizing marginalized groups for treatment with a drug like remdesivir would also be unethical because it would entail disproportionately exposing these groups to a therapeutic that may or not be beneficial [@doi:10.1080/15265161.2020.1795538].
-On the other hand, given that the drug is one of the most promising treatments available for many patients, using a framework that tacitly feeds into structural biases would also be unethical.
+Prioritizing marginalized groups for treatment with a drug like remdesivir would have been unethical because it would entail disproportionately exposing these groups to a therapeutic for which teh benefits and risks were not yet established [@doi:10.1080/15265161.2020.1795538].
```

Comment:
```suggestion
Prioritizing marginalized groups for treatment with a drug like remdesivir would have been unethical because it would entail disproportionately exposing these groups to a therapeutic for which the benefits and risks were not yet established [@doi:10.1080/15265161.2020.1795538].
```
```suggestion
Prioritizing marginalized groups for treatment with a drug like remdesivir would have been unethical because it would entail disproportionately exposing these groups to a therapeutic for which teh benefits and risks were not yet established [@doi:10.1080/15265161.2020.1795538].
```